src/constants.iced

```diff
@@ -36,6 +36,7 @@ exports.constants = constants =
   http_timeout : 15*1000 # give up after 15 seconds....
   short_id_bytes : 27
   shortest_pgp_signature : 100 # can't have a PGP signature shorter than this...
+  critical_clock_skew_secs : 60*60 # if you clock is an hour off, we won't accept the sig
```

Comment:
This seems like it could sit right on the edge of the daylight savings time delta, so that bugs related to that turn into random-looking race conditions instead of failing cleanly. Maybe we should use something that isn't an even multiple of an hour?